Admin/FaqAdmin.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Grossum\FaqBundle\Admin;
+
+use Sonata\AdminBundle\Admin\Admin;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Symfony\Component\Form\Extension\Core\Type as FormType;
```

Comment:
please, change this namespase to `IntegerType`. Don't use alias for `Type` folder.